### PR TITLE
Fix version sort order and add build_version block

### PR DESCRIPTION
**Pull request by ericholscher**

### CODE REVIEW

**Comment by agjohnson** [comment]
What is the intended effect of this PR?

**Comment by ericholscher** [comment]
Version sorting in the proper places, and the Version object change makes branches sort under numbered versions.

File: readthedocs/projects/version_handling.py [hunk 1]
```diff
@@ -127,7 +127,7 @@ def comparable_version(version_string):
         elif version_string == STABLE_VERBOSE_NAME:
             comparable = Version('9999.0')
         else:
-            comparable = Version('999.0')
+            comparable = Version('0.01')
```
Comment (agjohnson) [inline readthedocs/projects/version_handling.py]: This change may warrant an update to the docstring above, as we are assuming it's less than all other versions.

Comment (ericholscher) [inline readthedocs/projects/version_handling.py]: Good call. Fixed.

**Pull request merged**